beacon-chain/sync/data_columns.go

```diff
@@ -48,6 +74,67 @@ func RequestDataColumnSidecarsByRoot(
 	p2p p2p.P2P,
 	ctxMap ContextByteVersions,
 	newColumnsVerifier verification.NewDataColumnsVerifier,
+) ([]blocks.VerifiedRODataColumn, error) {
+	log := log.WithFields(logrus.Fields{
+		"blockRoot":          fmt.Sprintf("%#x", block.Root()),
+		"dataColumnsToFetch": dataColumnsToFetch,
+	})
+	log.Debug("Attempting to fetch or reconstruct data columns")
+
+	// First, try to request the columns directly.
+	sidecars, err := OnlyRequestDataColumnSidecarsByRoot(
```

Comment:
Nit: All arguments could be written on the same line (which is not *so* long).

Reply:
Proposal of renaming:

- `RequestDataColumnSidecarsByRoot` ==> `GetDataColumnSidecarsByRoot`
- `OnlyRequestDataColumnSidecarsByRoot` ==> `RequestDataColumnSidecarsByRoot`

The word `Get` is high level enough not to specify how the sidecars are obtained.

Reply:
43a6e16954522054d5e030d4cd6cd77dbafc6493